Programmers/230914_귤_고르기/kwaksj329.js

```diff
@@ -0,0 +1,20 @@
+function solution(k, tangerine) {
+  let answer = 0;
+  let tangerineSum = tangerine.reduce((acc, curr) => {
+    curr in acc ? (acc[curr] += 1) : (acc[curr] = 1);
```

Comment:
```suggestion
acc[curr] = (acc[curr] ?? 0) + 1;
```
이렇게도 쓸 수 있을 것 같아요 ㅎㅎ